spyder/utils/stylesheet.py

```diff
@@ -396,8 +438,212 @@ def set_stylesheet(self):
             right='-3px' if WIN else '-1px'
         )
 
+        # Make scroll button icons smaller on Windows and Mac
+        if WIN or MAC:
+            css[f'QTabBar{self.OBJECT_NAME} QToolButton'].setValues(
+                padding='9px',
```

Comment:
This is for the Editor/IPython console tabs.

Reply:
Using `5px` here I got:

![image](https://github.com/spyder-ide/spyder/assets/16781833/b804051d-37ad-499d-a73b-4c8f66cf934b)
![image](https://github.com/spyder-ide/spyder/assets/16781833/349b8f3f-f150-4873-b9b5-0d5c8b571ec6)

Reply:
Ok, thanks for the suggestion @dalthviz.

@conradolandia, what do you think about the size of these arrows? Do they look to you? I find them a bit too big, but I don't have a strong preference.

Reply:
I thinkk they look too big, yes. They should not be taller than the hamburger icon.

Reply:
> They should not be taller than the hamburger icon.

Yep, I agree.

@dalthviz, I selected the arrows size to match the one of the other icons in the toolbar. Perhaps we could make them two pixels bigger so they don't appear too small, but I'd say no more than that.

Reply:
Checking different values I got:

* `8px`:
![image](https://github.com/spyder-ide/spyder/assets/16781833/5e0725db-24ed-4ec6-b2a8-aa8fbf2fddd8)
![image](https://github.com/spyder-ide/spyder/assets/16781833/4460907d-a974-432a-96d1-672feb412553)

* `7px`:

![image](https://github.com/spyder-ide/spyder/assets/16781833/1c74ecde-c7e4-4083-b752-1a7725f34988)
![image](https://github.com/spyder-ide/spyder/assets/16781833/40ccd9bc-4565-4ce2-b4ca-02506c8cafcf)

Reply:
Thanks for the suggestions @dalthviz! I'll discuss them with @conradolandia in our UX meeting and I'll get back to you.

Reply:
@conradolandia mentioned he prefers `7px` here.